server/src/main/java/com/talkka/server/user/dao/UserEntity.java

```diff
@@ -0,0 +1,74 @@
+package com.talkka.server.user.dao;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
```

Comment:
```suggestion
@Entity(name = "users")
```
위와 같은 형태로 수정하면 `@Table` 어노테이션을 사용하지 않아도 됩니다.

Reply:
감사합니다. 해당 내용 확인하여 수정하겠습니다!